corehq/util/workbook_reading/adapters/xls.py

```diff
@@ -1,14 +1,22 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```

Comment:
I keep undoing this particular change when isort does this to my imports, because I vaguely remember @nickpell saying that it was useful to have these on separate lines, for scripting or grepping or something. I was hoping we could get isort to "force_single_line" just for `__future__` imports, and use "multi_line_output = 3" for everything else, but I don't think that's an option.

If this change is fine then I'll stop undoing it in my own code, and if it's not, consider this comment a flag.

Reply:
I think the scripting only cares about having ```print_function``` on a separate line, so this change is fine.